api/v1alpha4/types.go

```diff
@@ -105,6 +105,9 @@ var (
 	// accessible AWS Classic ELB scheme.
 	ClassicELBSchemeInternetFacing = ClassicELBScheme("internet-facing")
 
+	// ClassicELBSchemeIncorrectInternetFacing was inaccurately used to define an internet-facing LB in v0.6 releases > v0.6.6 and v0.7.0 release
```

Comment:
```suggestion
	// [Deprecated] ClassicELBSchemeIncorrectInternetFacing was inaccurately used to define an internet-facing LB in v0.6 releases > v0.6.6 and v0.7.0 release
```

Might as well do an immediate deprecation.